src/tools/vmap4_extractor/mpq_libmpq04.h

```diff
@@ -66,6 +66,7 @@ class MPQArchive
 
 private:
     void close();
+    bool isOpened() { return std::find(gOpenArchives.begin(), gOpenArchives.end(), this) != gOpenArchives.end(); }
```

Comment:
Please move isOpened method to .cpp file - gOpenArchives is not declared in the scope of this header